docs/sql/integer.md

```diff
@@ -10,6 +10,9 @@ The legal operations are `+` (plus, unary and binary), `-` (minus,
 unary and binary), `*` (multiplication), `/` (division), `%`
 (modulus).
 
+The modulo operator treats modulo by -1 (`x % -1`) as a special
```

Comment:
This is not really a special case. What should be here is a definition of what happens for negative numbers. Just showing the result for 8 % 3, 8 % -3, -8 % 3 and -8 % -3 should be enough.